src/library/Uno.Themes.WinUI.Markup/Uno.Themes.WinUI.Markup.csproj

```diff
@@ -22,4 +22,8 @@
 	</PropertyGroup>
 
 	<Import Project="..\..\..\winappsdk-workaround.targets" />
+	<Target Name="_UnoThemesRunGen" BeforeTargets="BeforeCompile">
+		<MSBuild Projects="..\Uno.Themes.WinUI.Markup.Generator\Uno.Themes.WinUI.Markup.Generator.csproj" />
+		<Exec Command='dotnet "..\Uno.Themes.WinUI.Markup.Generator\bin\Debug\net7.0\Uno.Themes.WinUI.Markup.Generator.dll"' />
```

Comment:
@Xiaoy312 I'm not going to push to the PR in case you already started continuing it locally, but `Debug` here likely needs to be replaced with `$(Configuration)`, that is probably why the command execution fails in CI.